JS Basics/Objects and Array methods/Problem_1/index.js

```diff
@@ -0,0 +1,32 @@
+function calculateDistance(x1, y1, x2, y2) {
+    return Math.sqrt((x2 - x1)**2 + (y2 - y1)**2)
+}
+
+function formTriangle(a, b, c) {
```

Comment:
This function returns boolean.
Its a good practice to name functions appropriately.
"formTriangle" leads to thinking that a triangle will be returned
use words like "can", "is", "should" etc. depending on the rest of the function name to indicate that it returns boolean